parm/config/config.fv3ic

```diff
@@ -1,19 +1,13 @@
 #!/bin/ksh -x
 
 ########## config.fv3ic ##########
-# Convert GFS initial conditions into FV3 initial conditions
 
 echo "BEGIN: config.fv3ic"
 
-# Task and thread configuration
-export wtime_fv3ic="00:30:00"
-export npe_fv3ic=1
-export npe_node_fv3ic=1
-export nth_fv3ic=${NTHREADS_CHGRES:-24}
-if [ $machine = HERA ]; then
-  export npe_fv3ic=4
-  export npe_node_fv3ic=4
-  export nth_fv3ic=1
-fi 
+export ORIGIN_ROOT=${BASE_CPLIC:-"/work/noaa/global/wkolczyn/noscrub/global-workflow/IC"}
```

Comment:
I'd prefer the default location to be the one on Hera since that is used for the prototypes, though I know we are working on Orion for this sprint.

Reply:
There are variables for this in each platform file in f/c-c, for example here: https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/workflow/platforms/hera.yaml#L39

Reply:
The orion.yaml has `BASE_CPLIC: "/work/noaa/marine/Partha.Bhattacharjee/IC_Dir"`. Should I add that or stick with Walter's for the Orion path?

I notice they are all personal account areas...we should move these files to a less personal area on all machines at some point...maybe a group account?

Reply:
Hera is the climate group account.  We don't have group accounts on orion.  Or do we now? We only actively run things on hera/orion.  Walter ported to DELL but prototypes aren't actually being run there really.

Reply:
We don't have a group account on Orion at the moment but I was told that they are coming. For now I am holding glopara group files in my personal account and will move it all to the group account when available. We have emc.glopara/glopara on WCOSS-Dell, Hera, and Jet.

I have added an if-block based on machine to set BASE_CPLIC for now.